services/core-api/app/extensions.py

```diff
@@ -40,8 +40,6 @@ def get_jwt_by_audience(aud):
 
 # Gold SSO
 jwtv2 = JwtManager(None, os.environ.get('JWT_OIDC_WELL_KNOWN_CONFIG'), None, 'RS256', None, None, os.environ.get('JWT_OIDC_AUDIENCE'), None, None, False, False, None, JWT_ROLE_CALLBACK, None)
-# Existing Keycloak for integration clients
-jwtv1 = JwtManager(None, os.environ.get('JWT_OIDC_WELL_KNOWN_CONFIG_V1'), None, 'RS256', None, None, os.environ.get('JWT_OIDC_AUDIENCE_V1'), None, None, False, False, None, JWT_ROLE_CALLBACK_V1, None)
```

Comment:
Can't the [def JWT_ROLE_CALLBACK_V1](https://github.com/bcgov/mds/blob/a4c24c5ce93a1ce6118596443881286db943e225/services/core-api/app/extensions.py#L17) be removed as well?

Reply:
It can indeed! Removed it